command/select.go

```diff
@@ -0,0 +1,163 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/peak/s5cmd/log/stat"
+	"github.com/peak/s5cmd/parallel"
+	"github.com/peak/s5cmd/storage"
+	"github.com/peak/s5cmd/storage/url"
+)
+
+var selectHelpTemplate = `Name:
+	{{.HelpName}} - {{.Usage}}
+
+Usage:
+	{{.HelpName}} [options] argument
+
+Options:
+	{{range .VisibleFlags}}{{.}}
+	{{end}}
+Examples:
+	01. Search for all JSON objects with the foo property set to 'bar' and spit them into stdout
+		 > s5cmd {{.HelpName}} --compression-type gzip --query "SELECT * FROM S3Object s WHERE s.foo='bar'" s3://bucket/*
+`
+
+var selectCommandFlags = []cli.Flag{
+	&cli.StringFlag{
+		Name:    "query",
+		Aliases: []string{"e"},
+		Usage:   "SQL expression to use to select from the objects",
+	},
+	&cli.StringFlag{
+		Name:  "compression-type",
```

Comment:
`--compression` might be better.

Reply:
Can you make  `--compression` flag optional?  (default: none)`select` command fails if we don't provide this flag

```
MissingRequiredParameter: CompressionType is required. Please check the service documentation and try again.
```